signingmodules/kaleidokms/internal/keymanagersigningmodule/config.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2025 Kaleido, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package keymanagersigningmodule
+
+import (
+	"github.com/kaleido-io/paladin/config/pkg/pldconf"
+)
+
+type Config struct {
+	HTTPConfig      *pldconf.HTTPClientConfig `json:"httpConfig,omitempty"`
+	KeystoreName    string                    `json:"keystoreName,omitempty"`
+	AutoKeyCreation bool                      `json:"autoKeyCreation"`
```

Comment:
I'm not sure why anything on the client side of the interface would be able to influence this 🤔